docs/desktop/wpf/fundamentals/xaml.md

```diff
@@ -1,11 +1,348 @@
+
+### Property element syntax
+
+For some properties of an object element, attribute syntax is not possible, because the object or information necessary to provide the property value cannot be adequately expressed within the quotation mark and string restrictions of attribute syntax. For these cases, a different syntax known as property element syntax can be used.
+
+The syntax for the property element start tag is `<TypeName.PropertyName>`. Generally, the content of that tag is an object element of the type that the property takes as its value. After specifying the content, you must close the property element with an end tag. The syntax for the end tag is `</TypeName.PropertyName>`.
+
+If an attribute syntax is possible, using the attribute syntax is typically more convenient and enables a more compact markup, but that is often just a matter of style, not a technical limitation. The following example shows the same properties being set as in the previous attribute syntax example, but this time by using property element syntax for all properties of the `Button`.
+
+[!code-xaml[XAMLOvwSupport#BlueRedButtonPE](~/samples/snippets/csharp/VS_Snippets_Wpf/XAMLOvwSupport/CSharp/Page1.xaml#blueredbuttonpe)]
+
+### Collection syntax
+
+The XAML language includes some optimizations that produce more human-readable markup. One such optimization is that if a particular property takes a collection type, then items that you declare in markup as child elements within that property's value become part of the collection. In this case a collection of child object elements is the value being set to the collection property.
+
+The following example shows collection syntax for setting values of the <xref:System.Windows.Media.GradientBrush.GradientStops%2A> property:
+
+```xaml
+<LinearGradientBrush>
+  <LinearGradientBrush.GradientStops>
+    <!-- no explicit new GradientStopCollection, parser knows how to find or create -->
+    <GradientStop Offset="0.0" Color="Red" />
+    <GradientStop Offset="1.0" Color="Blue" />
+  </LinearGradientBrush.GradientStops>
+</LinearGradientBrush>
+```
+
+### XAML content properties
+
+XAML specifies a language feature whereby a class can designate exactly one of its properties to be the XAML _content_ property. Child elements of that object element are used to set the value of that content property. In other words, for the content property uniquely, you can omit a property element when setting that property in XAML markup and produce a more visible parent/child metaphor in the markup.
+
+For example, <xref:System.Windows.Controls.Border> specifies a _content_ property of <xref:System.Windows.Controls.Decorator.Child%2A>. The following two <xref:System.Windows.Controls.Border> elements are treated identically. The first one takes advantage of the content property syntax and omits the `Border.Child` property element. The second one shows `Border.Child` explicitly.
```

Comment:
what is %2A for?

Reply:
It's a * in HTML format, some links to API need them, I'm unsure what the rules are about it but it's very common to see.

Reply:
ok cool, i'll ignore those then :)